src/integ-test/bdd-test/features/primitives/paVerifyPaymentNoticeRes_syntax_OK.feature

```diff
@@ -0,0 +1,38 @@
+ Feature: syntax checks for paVerifyPaymentNoticeRes - OK
+ 
+ Background: Given systems up   
+    And PA new version
+    And valid verifyPaymentNoticeReq soap-request    

```

Comment:
it should be a `paVerifyPaymentNoticeReq` soap request

Reply:
And I would insert even an other then step with `paVerifyPaymentNoticeRes` response template

Reply:
it should be a paVerifyPaymentNoticeReq soap request >> In our test it has been considered as precondition a valid and correct 'verifyPaymentNoticeReq', since the paVerifyPaymentNotice isn't directly managed by us. If according to you it's better to consider a valid paVerifyPaymentNoticeReq it should be changed the payload, too.